src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const table = document.body.firstElementChild;
+
+for (let i = 0; i < table.rows.length; i++) {
+  const columnToClone = table.rows[i].cells[1];
+  const tagToCreate = document.createElement(columnToClone.tagName);
+
+  tagToCreate.textContent = columnToClone.textContent;
+
+  table.rows[i].cells[4].before(tagToCreate);
```

Comment:
The index of 'cells[4]' may lead to an out of bounds error if the table does not have 5 columns. It is better to use 'cells.length - 1' to always target the last column.